tools/i18n/build.py

```
@@ -0,0 +1,373 @@
+
+    if output_dir:
+        out_dir = Path(output_dir)
+    else:
+        out_dir = Path(tempfile.mkdtemp(prefix=f'wled_i18n_{locale}_'))
+        print(f"[i18n] No --output-dir specified, using temp: {out_dir}")
+
+    out_dir.mkdir(parents=True, exist_ok=True)
+
+    if out_dir.resolve() == src_dir.resolve():
+        print(f"WARNING: Output dir equals source dir ({src_dir}).", file=sys.stderr)
+        print(f"  English source files will be overwritten!", file=sys.stderr)
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)